Partial revert of #40431

SgtxRootbeer opened the pull request:
:cl: Sgt. Rootbeer
Balance: Mecha and Aux base turret's plasma cutters have been buffed back to full power.
/:cl:
This is because the intent in #40431 was to nerf the handheld plasma cutters so as to prevent spam against aliens and blobs. Because it is nearly impossible to spam RIPLEYS, and definetly impossible to spam aux base turrets against blobs and xenos the nerf against them was out of scope of the PR, and wholly unnecessary.

Yes, technically now both of them are capable of doing full damage in full presure enviornments, but again, ripleys are weak, and the aux base can't even be used offensively by a player.

If anyone wants to fix this correctly by re-implementing the pressure change only for the mecha and the base, then by all means, go ahead. For now, I'd consider this good.

Commit:
undo nerf

buffs back to 40/24

SgtxRootbeer commented:
@HydroFloric this only reverts the damage nerf, so the pc can fight Goliath's and whatnot just fine.

ExcessiveUseOfCobblestone commented:
Is there a reason you chose the 0 effort PR over your "proper" solution?

Barhandar commented:
@HydroFloric the original PR also removed ranged dismemberment (without adding a changelog for it).

ExcessiveUseOfCobblestone commented:
```DM
 /obj/item/projectile/plasma
// blah blah blah
        var/pressure_check_scaling = FALSE
/obj/item/projectile/plasma/Initialize()	
	. = ..()	
	if(pressure_check_scaling && !lavaland_equipment_pressure_check(get_turf(src)))	
		name = "weakened [name]"	
		damage = damage * pressure_decrease	
		pressure_decrease_active = TRUE

// Under both the mech and turret
        var/pressure_check_scaling = FALSE
```

optimumtact commented:
As Cobby said

SgtxRootbeer commented:
@ExcessiveUseOfCobblestone @optimumtact ill do that, but i'll also re-add dismemberment too.

i considered it balanced regardless because the PC doesnt have dismemberment anymore.

ishitbyabullet commented:
https://github.com/tgstation/tgstation/pull/40578

I personally like Imsxz's PR more.

Commit:
As oranges required

because nobody can figure out how the code works, and oranges cant actually read what he's approving

SgtxRootbeer commented:
now it uses cobby's pressure-check code.
i also reverted the dismemberment removal for the aux base and the mecha cutter. will they use it? no. am i doing it simply for the principle of it? yes.

SgtxRootbeer commented on code/modules/projectiles/projectile/special/plasma.dm:
```diff
@@ -27,13 +34,17 @@
 	mine_range = 3
-
+	dismemberment = 20
+	pressure_check_scaling = TRUE
```
also, cobby  set this to false in his recommended code, but I set it to true. I believe it was a typo.

SgtxRootbeer commented:
got any reason behind this, aside from potentially hurt feelings?

optimumtact commented:
`I personally like Imsxz's PR more.`
